web/src/pages/mgmt/deadurls.tsx

```diff
@@ -15,9 +16,50 @@ const DeadUrlsPage = (props: Props): ReactElement => {
   const [isAuthed, setIsAuthed] = useState<boolean>(false);
   const [password, setPassword] = useState<string>("");
 
+  const generateContent = (): string => {
+    let deadLinkFileOutput = "Possibly Blocked Or Deadlinks \n\n";
```

Comment:
Do we have a config to store this?

Reply:
We don't because this isn't actually user-facing so we don't need to set it so content can change it (because it doesn't matter)